.github/workflows/lint_mrtk3.yaml

```diff
@@ -0,0 +1,28 @@
+name: Lint (MRTK3)
+permissions:
+  statuses: write
+  checks: write
+
+on:
+  push:
+    branches-ignore: ["mrtk3"]
+  pull_request_target:
```

Comment:
Using `pull_request_target` means this won't contain any of the PR changes in the `checkout` step, right?

Reply:
I have... no idea... 

The lint-action docs imply that this will work. 

![image](https://user-images.githubusercontent.com/5544935/194134751-1469b915-428f-4c3c-bc9d-65b953d957ad.png)

Reply:
Well that definitely didn't work